WORKSPACE

```diff
@@ -74,6 +74,10 @@ py_library(
     url = "https://pypi.python.org/packages/source/m/mock/mock-1.0.1.tar.gz",
 )
 
+load("//k8s:kubeval_repos.bzl", "kubeval_repositories")
+
+kubeval_repositories()
```

Comment:
Feels like we should be passing the K8s versions we care about here?

What I had in my head was something like:
```python
WORKSPACE
k8s_schema(
   name = "foo_1_7",
   version = "1.7",
)

BUILD
kubeval_test(
    ...
   schemas = ["@foo_1_7"],
)
```

If/When this moves into `k8s_object` as the `build` step, then it's be awesome if the aliasing logic could handle this too.

Reply:
Done.